{"identifier":{"url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"topicSections":[{"identifiers":["doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/failureReason","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/helpAnchor","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/recoverySuggestion"],"generated":true,"title":"Instance Properties"}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openai\/apierror\/localizederror-implementations"]}],"metadata":{"roleHeading":"API Collection","modules":[{"name":"OpenAI"}],"title":"LocalizedError Implementations","role":"collectionGroup"},"schemaVersion":{"minor":3,"major":0,"patch":0},"kind":"article","hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError"]]},"references":{"doc://OpenAI/documentation/OpenAI":{"type":"topic","abstract":[],"role":"collection","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","title":"OpenAI","url":"\/documentation\/openai","kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError/errorDescription":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror\/errordescription","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription"},"doc://OpenAI/documentation/OpenAI/APIError/recoverySuggestion":{"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"title":"recoverySuggestion","kind":"symbol","url":"\/documentation\/openai\/apierror\/recoverysuggestion","type":"topic","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/recoverySuggestion","abstract":[],"role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError/helpAnchor":{"url":"\/documentation\/openai\/apierror\/helpanchor","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/helpAnchor","abstract":[],"type":"topic","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}],"kind":"symbol","title":"helpAnchor"},"doc://OpenAI/documentation/OpenAI/APIError":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror","title":"APIError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"APIError","kind":"identifier"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","navigatorTitle":[{"kind":"identifier","text":"APIError"}]},"doc://OpenAI/documentation/OpenAI/APIError/failureReason":{"abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror\/failurereason","title":"failureReason","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/failureReason"}}}